builtins/amp-video.md

```diff
@@ -82,9 +83,10 @@ Similar to the `video` tag `controls` attribute - if present, the browser offers
 
 If present, will automatically loop the video back to the start upon reaching the end.
 
-**muted**
+**muted (deprecated)**
```

Comment:
How about starting on conversation on whatwg/html to clarify what `muted` attribute implies for the UA. The way iOS is interpreting it is very restrictive. The spec is not very clear either: "muted - Whether to mute the media resource by default". It does not say if UA should still allow the user to unmute a muted video or not.